app/routes/company/components/CompanyDetail.tsx

```diff
@@ -23,224 +28,138 @@ type Props = {
+      <ContentSection>
+        <ContentMain>
+          <i>
+            <DisplayContent content={company.description} />
+          </i>
+          <h3 className={styles.sectionHeader}>Kommende arrangementer</h3>
+          <ListEvents
+            events={upcomingEvents}
+            noEventsMessage="Ingen kommende arrangementer"
+            loggedIn={loggedIn}
+            eventStyle="extra-compact"
+          />
 
-      {insertInfoBubbles(company)}
+          {oldEvents.length > 0 && (
+            <Button onClick={() => setViewOldEvents(!viewOldEvents)}>
+              {viewOldEvents
+                ? 'Skjul tidligere arrangementer'
+                : 'Vis tidligere arrangementer'}
+            </Button>
+          )}
+          {viewOldEvents && (
+            <>
+              <h3 className={styles.sectionHeader}>Tidligere arrangementer</h3>
+              <ListEvents
+                events={oldEvents}
+                noEventsMessage="Ingen tidligere arrangementer"
+                loggedIn={loggedIn}
+                eventStyle="extra-compact"
+              />
+            </>
+          )}
+          {viewOldEvents && showFetchMoreEvents && (
+            <Flex style={{ justifyContent: 'center' }}>
+              <Icon
+                name="chevron-down-circle-outline"
+                size={35}
+                onClick={fetchMoreEvents}
+                style={{ cursor: 'pointer' }}
+              />
+            </Flex>
+          )}
 
-      <h3
-        style={{
-          marginTop: '20px',
-        }}
-      >
-        Bedriftens arrangementer
-      </h3>
-      {companyEvents.length > 0 ? (
-        <CompanyEvents
-          companyEvents={companyEvents}
-          showFetchMoreEvents={showFetchMoreEvents}
-          fetchMoreEvents={fetchMoreEvents}
-          loggedIn={loggedIn}
-        />
-      ) : (
-        <i>Ingen arrangementer.</i>
-      )}
-      <h3
-        style={{
-          marginTop: '20px',
-        }}
-      >
-        Bedriftens jobbannonser
-      </h3>
-      {joblistingsList.length > 0 ? (
-        <table className={styles.companyEventTable}>
-          <thead>
-            <tr>
-              <th>Tittel</th>
-              <th>Jobbtype</th>
-              <th>Klassetrinn</th>
-              <th>Søknadslenke</th>
-            </tr>
-          </thead>
-          <tbody>{joblistingsList}</tbody>
-        </table>
-      ) : (
-        <i>Ingen jobbannonser.</i>
-      )}
+          <h3 className={styles.sectionHeader}>Jobbannonser</h3>
+          {joblistings.length > 0 ? (
+            joblistings.map((joblisting) => (
+              <JoblistingItem key={joblisting.id} joblisting={joblisting} />
+            ))
+          ) : (
+            <i>Ingen tilgjengelige jobbannonser</i>
+          )}
+        </ContentMain>
+        <ContentSidebar>
+          {companyInfo.map(
+            (info) =>
+              info.text && (
+                <span key={info.text} className={styles.iconWithText}>
```

Comment:
Could turn this into a `<Flex />` component since `styles.iconWithText` only adds flex styling.